src/index.ts

```diff
@@ -1,16 +1,91 @@
+export { Invariant } from './invariant'
+export { FungibleToken, TokenMetaData } from './fungible-token'
+export { Network } from './network'
+export {
+  calculateSqrtPrice,
+  calculateTick,
+  getLiquidityByX,
+  getLiquidityByY,
+  getMaxChunk,
+  getMaxSqrtPrice,
+  getMinSqrtPrice,
+  isTokenX,
+  bitPositionToTick,
+  priceToSqrtPrice,
+  calculateFee,
+  calculatePriceImpact,
+  calculateSqrtPriceAfterSlippage,
+  // calculateTickDelta,
+  calculateTokenAmounts,
+  sqrtPriceToPrice,
+  getMinTick,
+  getMaxTick,
+  toFeeGrowth,
+  toFixedPoint,
```

Comment:
we are using `fixedPoint` only to calculate `sqrtPrice` in intermediate operations in `calculateSqrtPrice`. It shouldn't be exported from SDK.